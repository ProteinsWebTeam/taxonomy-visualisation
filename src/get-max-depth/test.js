import getMaxDepth from '.';

test('getMaxDepth should return valid values', () => {
  const depthCountSets = [
    [1, 2, 3, 4, 5, 6],
    [1, 2, 3, 4, 50, 6],
    [1, 2, 20, 4, 50, 6],
    [1, 2],
    [1, 2, 100, 1000],
  ];
  const maxCounts = [1, 2, 10, 100];
  for (const depthCounts of depthCountSets) {
    for (const maxCount of maxCounts) {
      expect(getMaxDepth(depthCounts, maxCount)).toMatchSnapshot();
    }
  }
});
