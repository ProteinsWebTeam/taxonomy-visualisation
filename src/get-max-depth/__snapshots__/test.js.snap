// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getMaxDepth should return valid values 1`] = `0`;

exports[`getMaxDepth should return valid values 2`] = `1`;

exports[`getMaxDepth should return valid values 3`] = `5`;

exports[`getMaxDepth should return valid values 4`] = `5`;

exports[`getMaxDepth should return valid values 5`] = `0`;

exports[`getMaxDepth should return valid values 6`] = `1`;

exports[`getMaxDepth should return valid values 7`] = `3`;

exports[`getMaxDepth should return valid values 8`] = `5`;

exports[`getMaxDepth should return valid values 9`] = `0`;

exports[`getMaxDepth should return valid values 10`] = `1`;

exports[`getMaxDepth should return valid values 11`] = `1`;

exports[`getMaxDepth should return valid values 12`] = `5`;

exports[`getMaxDepth should return valid values 13`] = `0`;

exports[`getMaxDepth should return valid values 14`] = `1`;

exports[`getMaxDepth should return valid values 15`] = `1`;

exports[`getMaxDepth should return valid values 16`] = `1`;

exports[`getMaxDepth should return valid values 17`] = `0`;

exports[`getMaxDepth should return valid values 18`] = `1`;

exports[`getMaxDepth should return valid values 19`] = `1`;

exports[`getMaxDepth should return valid values 20`] = `2`;
